src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -114,7 +114,7 @@ msrest==0.7.1
 msrestazure==0.6.4
 oauthlib==3.2.2
 packaging==23.0
-paramiko==2.10.1
+paramiko==2.11.0
```

Comment:
Perhaps we can bump to 3.0.0?

Reply:
JFYI azure-cli-core currently requires a version `<3.0.0`

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L59

Maybe @marstr remembers [why](https://github.com/Azure/azure-cli/pull/9668)

Reply:
Wow, this is a blast from the past! Unfortunately, I can't remember the rationale behind this restriction. Best of luck.

Reply:
`paramiko<3.0.0` is introduced in https://github.com/Azure/azure-cli/pull/9668. It looks like a general update and pin it to version 2.xx to prevent breaking changes.

I update it to 3.0.0 in [this branch](https://github.com/Azure/azure-cli/tree/fix-blowfish-deprecation-warnings), and the tests are passed.

Reply:
@StrawnSC Could you please update paramiko to 3.0.0 and change this line to `<4.0.0` https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L59